versions/draft-3/SPEC.md

```diff
@@ -636,6 +636,16 @@ $document = ($import | $task | $workflow)+
 
 `$document` is the root of the parse tree and it consists of one or more import statement, task, or workflow definition
 
+## Versioning
+
+For portability purposes it is critical that WDL documents be versioned so an engine knows how to process it. A WDL document is versioned with a `version` statement, for eample
+
+```wdl
+version draft-3
+```
+
+All WDL documents must have exactly one `version` statement. Any WDL document which does not have a `version` field may be treated as `draft-2`, however an implementation may choose to also support non-backwards incompatible changes which have made subsequent to `draft-2`.  
```

Comment:
if the version statement isn't "commented" it would be impossible to version draft-2 WDL files.

Reply:
Why would you need to version a `draft-2` document? If there's no version it **is** draft-2.

Reply:
Well, modulo some small set of WDLs in the wild which have been created since `draft-3` was created. I'd put forth that the people writing such WDLs are active enough that they'll manage to be able to add `draft-3` to their stuff.

Reply:
It would be nice to be able to be explicit....I don't need to.